client/ayon_core/tools/publisher/publish_report_viewer/widgets.py

```diff
@@ -538,6 +606,7 @@ def _on_plugin_change(self, *_args):
                 plugin_ids.add(index.data(ITEM_ID_ROLE))
 
         self._logs_text_widget.set_plugin_filter(plugin_ids)
+        self._class_info_widget.set_plugin_filter(plugin_ids)
```

Comment:
Preferably these would only end up triggering 'their logic' if they are currently visible. In these cases both these widgets maybe aren't doing too much complicated work yet and thus it might be quite fast - but as we expand on these UIs we might want to, not necessarily in this PR, just pass on 'marking the widgets dirty for a refresh' and only actually trigger the refresh code if the widget is visible?

Reply:
Agree, but not in this PR please.